backend/src/main/java/com/javahelp/backend/accountdeletion/DeleteManager.java

```diff
@@ -0,0 +1,38 @@
+package com.javahelp.backend.accountdeletion;
+
+import com.javahelp.backend.data.IUserStore;
+import com.javahelp.model.user.User;
+import com.javahelp.model.user.UserPassword;
+
+/**
+ * A class for managing an account deletion action.
+ */
+class DeleteManager {
+
+    /**
+     * The {@link IUserStore} to use.
+     */
+    private final IUserStore userStore;
+
+    /**
+     * Constructs a {@link DeleteManager} instance.
+     *
+     * @param userStore: the {@link IUserStore} used to store users.
+     */
+    protected DeleteManager(IUserStore userStore) {
```

Comment:
Similar to the class, the constructor must also be public for the same reasons.

Reply:
Should I make the delete method public as well?

Reply:
> Should I make the delete method public as well?

I think so.